packages/integration_test/analysis_options.yaml

```diff
@@ -0,0 +1,5 @@
+include: ../analysis_options.yaml
+
+analyzer:
+  exclude:
+    - "test_fixes/**"
```

Comment:
Nit: should this have a new line at the end of the file?

Reply:
Why yes, yes it should! Good catch!

Thanks, and fixed!